src/utils/http/errors/FetchError.js

```diff
@@ -0,0 +1,12 @@
+class FetchError extends Error {
+  constructor(error) {
+    super(error);
+
+    Object.setPrototypeOf(this, new.target.prototype);
```

Comment:
What does this do?

Reply:
I have no idea and I haven't found the link where I copied it from. Removed